execution_engine_testing/tests/src/test/contract_api/get_call_stack.rs

```diff
@@ -268,6 +269,7 @@ fn assert_each_context_has_correct_call_stack_info_module_bytes(
             EntryPointType::Session => {
                 builder.get_call_stack_from_session_context(&stored_call_stack_key)
             }
+            _ => unreachable!(),
```

Comment:
Can we handle this differently than an `unreachable`

Reply:
Fixed